### PR TITLE
revoke confirmation

### LINKED ISSUE #1605: Revoke should confirm

### PR DESCRIPTION
**Pull request by na3d**
Closes #1605 
screen shot 2017-06-06 at 4 36 18 pm

### CODE REVIEW

File: src/profile/integrations/components/AuthorizedApplication.js [hunk 1]
```diff
@@ -36,23 +36,28 @@ export default class AuthorizedApplication extends Component {
         onCancel={() => dispatch(hideModal())}
       >
         <p>
-          Are you sure you want to
-          revoke <strong>{name}</strong>'s access?
+          Are you sure you want to revoke this token's access,
+          created {created} by <strong>{name}</strong>?
```
Comment (eatonphil) [inline src/profile/integrations/components/AuthorizedApplication.js]: TimeDisplay plz?

File: src/profile/integrations/components/AuthorizedApplication.js [hunk 2]
```diff
@@ -21,16 +23,41 @@ export default class AuthorizedApplication extends Component {
...
+      </ConfirmModalBody>
+    ));
+  }
+
   render() {
-    const { label, scopes, id, clientId, expires } = this.props;
+    const { label, scopes, id, clientId, expires, created } = this.props;
     const icon = clientId ? `${API_ROOT}/account/clients/${clientId}/thumbnail` : '';
     const expireValue = <TimeDisplay time={expires} />;
+    const createValue = <TimeDisplay time={created} />;
```
Comment (na3d) [inline src/profile/integrations/components/AuthorizedApplication.js]: @eatonphil It is here^

Comment (eatonphil) [inline src/profile/integrations/components/AuthorizedApplication.js]: Oh gotcha.

Comment (na3d) [inline src/profile/integrations/components/AuthorizedApplication.js]: I can move it into the function, but felt that this seems more organized.

**Pull request merged**